test/user.test.js

```diff
@@ -91,7 +91,7 @@ describe('User', function() {
         assert.equal(err.details.context, User.modelName);
         assert.deepEqual(err.details.codes.email, [
           'presence',
-          'format.null'
+          'custom'
```

Comment:
I'm not sure how to do this. The Validatable api in loopback-datasource-juggler/lib/validations.js doesn't seem to allow override of the validator name. Should i move the logic in a new Validatable.validateEmail function there and then simply use it in the user model ?